sound/soc/sof/intel/hda-dai.c

```diff
@@ -447,37 +447,6 @@ static struct snd_soc_cdai_ops sof_probe_compr_ops = {
 #endif
 #endif
 
-static int ssp_dai_config_update(struct snd_pcm_substream *substream, struct snd_soc_dai *dai,
```

Comment:
@juimonen is this correct?

Reply:
> @juimonen is this correct?

I think we already have sof_widget_setup() (it calls sof_dai_config_setup()) called at sof_widget_list_setup(), we don't need an extra duplicate ssp_dai_config_update()? anyway, can you share how to test it @juimonen ?

Reply:
we tested this with @keyonjie: in ssp multi config topology the correct ssp config is chosen and sent to dsp before stream starts -> so this part LGTM